monkey/infection_monkey/dropper.py

```diff
@@ -166,16 +162,13 @@ def start(self):
             dest_path = self._config["destination_path"]
             # In Linux, we need to change the directory first, which is done
             # using thw `cwd` argument in `subprocess.Popen` below
-            # TODO: Replace all of this string templating with a function that accepts
-            #       the necessary parameters and returns a list of arguments.
-            monkey_cmdline = (
-                MONKEY_CMDLINE_LINUX % {"monkey_filename": dest_path.split("/")[-1]}
-                + monkey_options
-            )
-            monkey_cmdline_split = shlex.split(monkey_cmdline)
+
+            monkey_cmdline = get_monkey_cmd_lines_linux(dest_path, monkey_options)
+
+            LOG.info("Commands of monkey cmdline_split %s", monkey_cmdline)
```

Comment:
I'm not totally sure what this log message is for. We log the command that was used after the call to `Popen()`, so I think this log message is redundant.